mkdocs.yml

```diff
@@ -55,13 +55,13 @@ pages:
   - Puppet Gluster: Administrator Guide/Puppet.md
   - RDMA Transport: Administrator Guide/RDMA Transport.md
   - GlusterFS iSCSI: Administrator Guide/GlusterFS iSCSI.md
-  - Configuring NFS-Ganesha server: Administrator Guide/NFS-Ganesha GlusterFS Intergration.md
+  - Configuring NFS-Ganesha server: Administrator Guide/NFS-Ganesha GlusterFS Integration.md
```

Comment:
If we correct above, we need to correct the file name here https://github.com/gluster/glusterdocs/blob/master/Administrator%20Guide/NFS-Ganesha%20GlusterFS%20Intergration.md